src/sparse/impl/KokkosSparse_sptrsv_symbolic_impl.hpp

```diff
@@ -320,9 +288,8 @@ void lower_tri_symbolic(TriSolveHandle& thandle, const RowMapType drow_map,
     Kokkos::deep_copy(dnodes_grouped_by_level, nodes_grouped_by_level);
     Kokkos::deep_copy(dnodes_per_level, nodes_per_level);
     Kokkos::deep_copy(dlevel_list, level_list);
-    if (stored_diagonal) Kokkos::deep_copy(diagonal_offsets, hdiagonal_offsets);
```

Comment:
@ndellingwood I am just curious. Why is it okay now to remove the diagonal_offsets? Is it not used anywhere?

Reply:
@vqd8a no, I had added this as a possible optimization to explore but I don't think I saw much improvement when testing (been quite awhile since I looked into it). I missed cleaning this out during a cleanup for a past PR